setup.py

```diff
@@ -62,11 +62,11 @@
     "flatten-json==0.1.6",
     "boto>=2.48.0, <2.49",
     "python-magic>=0.4.13, <0.4.14",
+    "validators<=0.14.2",
```

Comment:
any reason to pick 0.14.2? I think by 0.16.0 release date, it should be good. https://libraries.io/pypi/validators. Also there is + can be removed.

Reply:
because versions between 0.14.2 and 0.17.0 are yanked? https://pypi.org/project/validators/#history

Reply:
make sense. + sign can be removed.

Reply:
where is the `+` sign?

Reply:
ignore, my bad